openpype/plugins/publish/validate_unique_subsets.py

```diff
@@ -9,8 +10,8 @@ class ValidateSubsetUniqueness(pyblish.api.ContextPlugin):
     """Validate all subset names are unique.
 
     This only validates whether the instances currently set to publish from
-    the workfile overlap one another for the asset + subset they are publishing
-    to.
+    the workfile overlap one another for the asset + subset + version number
+    they are publishing to.
```

Comment:
I feel like matching asset+subset but different versions are still a "unique special" case in publishing and might still warrant their own warning message. Those are currently not implemented and having those logged as warnings maybe help identifying instances that 'overrwite' each other but are solely lacking a matching `instance.data["version"]`?

Reply:
Yes, that would make sense to let know user about the case. Here is how I understand the situation in code eef230d81e3c40f0355cb2aacf6fb83774e988a8. Would that be close to what you suggest?